mysql/infoschema.go

```diff
@@ -304,6 +310,120 @@ func getConstraints(conv *internal.Conv, db *sql.DB, table schemaAndName) ([]str
 	return primaryKeys, m, nil
 }
 
+// getForeignKeys return list all the foreign keys constraints.
+// MySQL supports cross-database foreign key constraints. We ignore
+// them because HarbourBridge works database at a time (a specific run
+// of HarbourBridge focuses on a specific database) and so we can't handle
+// them effectively.
+func getForeignKeys(conv *internal.Conv, db *sql.DB, table schemaAndName) (foreignKeys []schema.ForeignKey, err error) {
+	q := `SELECT k.REFERENCED_TABLE_NAME,k.COLUMN_NAME,k.REFERENCED_COLUMN_NAME
+		FROM INFORMATION_SCHEMA.TABLE_CONSTRAINTS AS t 
+		INNER JOIN INFORMATION_SCHEMA.KEY_COLUMN_USAGE AS k 
+			ON t.CONSTRAINT_NAME = k.CONSTRAINT_NAME 
+			AND t.CONSTRAINT_SCHEMA = k.CONSTRAINT_SCHEMA 
+			AND t.TABLE_NAME = k.TABLE_NAME 
+			AND k.REFERENCED_TABLE_SCHEMA = k.TABLE_SCHEMA
+		WHERE k.TABLE_SCHEMA = ? 
+			AND k.TABLE_NAME = ? 
+			AND t.CONSTRAINT_TYPE = "FOREIGN KEY" 
+		ORDER BY
+			k.REFERENCED_TABLE_NAME,
+			k.ordinal_position;`
+	rows, err := db.Query(q, table.schema, table.name)
+	if err != nil {
+		return nil, err
+	}
+	defer rows.Close()
+	var col, refCol, refTable string
+	mCols := make(map[string][]string)
+	mRefCols := make(map[string][]string)
+	for rows.Next() {
+		err := rows.Scan(&refTable, &col, &refCol)
+		if err != nil {
+			conv.Unexpected(fmt.Sprintf("Can't scan: %v", err))
+			continue
+		}
+		mCols[refTable] = append(mCols[refTable], col)
+		mRefCols[refTable] = append(mRefCols[refTable], refCol)
+	}
+	for k, v := range mCols {
+		foreignKeys = append(foreignKeys, schema.ForeignKey{Columns: v, ReferTable: k, ReferColumns: mRefCols[k]})
+	}
+	return foreignKeys, nil
+}
+
+// // getForeignKeys return list all the foreign keys constraints.
```

Comment:
I think we can delete this commented block of code now.

Reply:
removed this commented block